Library/Homebrew/utils/github/actions.rb

```diff
@@ -110,7 +111,11 @@ def to_s
           end
         end
 
-        metadata << ",title=#{Actions.escape(@title)}" if @title
+        if @title
+          metadata << (@file ? "," : " ")
```

Comment:
This should probably go into an array which we `join(",")`

Reply:
Yea, good point.